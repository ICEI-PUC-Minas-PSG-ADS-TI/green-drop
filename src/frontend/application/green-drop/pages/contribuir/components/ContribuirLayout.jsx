// /contribuir/components/ContribuirLayout.jsx
import React from 'react';
import { View, StyleSheet } from 'react-native';
import ProgressHeader from './ProgressHeader';
import BottomBar from '@/components/BottomBar';

const ContribuirLayout = ({ currentStep, children, style }) => (
  <View style={styles.container}>
    <ProgressHeader currentStep={currentStep} />
    <View style={[styles.content, style]}>
      {children}
    </View>
    <BottomBar/>
  </View>
);

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
  content: {
    flex: 1,
    
    marginBottom: 60,
  },
});

export default ContribuirLayout;