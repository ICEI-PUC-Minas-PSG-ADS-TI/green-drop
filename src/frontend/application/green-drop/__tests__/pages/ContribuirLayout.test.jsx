import React from 'react';
import { render } from '@testing-library/react-native';
import renderer from 'react-test-renderer';
import ContribuirLayout from '../../pages/contribuir/components/ContribuirLayout.jsx';

describe('<ContribuirLayout />', () => {
it('renderiza corretamente', () => {
render(<ContribuirLayout />);
});

it('snapshot de ContribuirLayout', () => {
const tree = renderer.create(<ContribuirLayout />).toJSON();
expect(tree).toMatchSnapshot();
});
});
