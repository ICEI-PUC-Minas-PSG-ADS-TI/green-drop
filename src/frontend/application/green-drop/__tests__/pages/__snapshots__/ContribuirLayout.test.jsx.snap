// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContribuirLayout /> snapshot de ContribuirLayout 1`] = `null`;
